src/containers/pages/InterventionPlan/UpdatePlan/CaseDetails/tests/index.test.tsx

```diff
@@ -49,6 +49,24 @@ describe('src/containers/pages/interventionPlan/updateplan/caseDetials', () => {
     expect(wrapper.find('Ripple').length).toEqual(0);
     expect(wrapper.text()).toMatchSnapshot(`Case Details`);
   });
+  it('renders nothing when eventId is null', async () => {
```

Comment:
@p-netm does this test that displayError was called with the expected error message?

Reply:
add that.